packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -666,7 +666,13 @@ export default class AwsS3Multipart extends UploaderPlugin {
 
         // NOTE This must be allowed by CORS.
         const etag = ev.target.getResponseHeader('ETag')
+        const location = ev.target.getResponseHeader('Location')
 
+        if (method === 'POST' && location === null) {
```

Comment:
```suggestion
        if (method?.toLowerCase() === 'POST'.toLowerCase() && location === null) {
```
`'post' === 'POST'` = false

Reply:
`==` instead of `===` on the location null check?